docs/static-rendering.md

```diff
@@ -0,0 +1,230 @@
+
+After configuring sku, the render entry needs to return the HTML required to create all the above files. The render entry is set via `entry.render` (default is `src/render.js`).
+
+**Example render entry**
+
+```js
+import React from 'react';
+import { renderToString } from 'react-dom/server';
+import App from './App';
+
+export default {
+  renderApp: ({ environment, site, route }) =>
+    renderToString(<App environment={environment} site={site} route={route} />),
+
+  renderDocument: ({ app, bodyTags, headTags }) => `
+    <!DOCTYPE html>
+    <html>
+      <head>
+        <meta charset="UTF-8">
+        <title>My Awesome Project</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        ${headTags}
+      </head>
+      <body>
+        <div id="app">${app}</div>
+        ${bodyTags}
+      </body>
+    </html>
+  `
+};
+```
+
+## renderApp
+
+The `renderApp` function should return your application as an HTML string (generally using `React.renderToString`). It can also return other values (e.g. extracted meta information, CSS styles, etc). Anything returned will be available in `renderDocument`. You should not render your whole HTML document in `renderApp`, only return the HTML that is generated by React.
+
+`renderApp` will be called once for each combination of settings in sku config. Specifically, `environment`, `site` & `route`.
+
+## renderDocument
+
+`renderDocument` is called immediately after `renderApp`. It gets passed all the same values as `renderApp` plus the following. It must return a full HTML document.
+
+- `app` - the value returned from the `renderApp` function
+- `headTags` - html tags to be placed in the head of the html
+- `bodyTags` - html tags to be placed at the end of the html body
+
+## Common use cases
+
+### TypeScript
+
+If your app uses TypeScript, sku provides the type definitions for the render entry.
+
+```ts
+import React from 'react';
+import { renderToString } from 'react-dom/server';
+import { Render } from 'sku';
+import App from './App';
+
+interface RenderContext {
+  html: string;
+  otherThing: number;
+}
+const skuRender: Render<RenderContext> = {
+  renderApp: () => ({
+    html: renderToString(<App />),
+    otherThing: 10
+  }),
+
+  renderDocument: ({ app, headTags, bodyTags }) => `
+    <!DOCTYPE html>
+    <html>
+      <head>
+        <meta charset="UTF-8">
+        <title>My Awesome Project</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        ${headTags}
+      </head>
+      <body>
+        <div id="app">${app.html}</div>
+        ${bodyTags}
+      </body>
+    </html>
+  `
+};
+
+export default skuRender;
+```
+
+### React Helmet
+
+React Helmet requires extra work after React is finished rendering to extract static meta information. This is also a common pattern used by some other libraries (e.g. [react-loadable](https://github.com/jamiebuilds/react-loadable), [emotion](https://emotion.sh/docs/ssr)).
```

Comment:
Be good to turn "React Helmet" into a link here as we've done with loadable and emotion